src/main/app/utils/featureToggles.ts

```diff
@@ -6,6 +6,21 @@ export class FeatureToggles {
     return FeatureToggles.isAnyEnabled(featureName)
   }
 
+  static hasAuthorisedFeature (feature: string, authorisedFeatures: string[]): boolean {
```

Comment:
Please remove. That alias function does not make sense to me. Use `hasAnyAuthorisedFeature` instead.